solidity/deploy/00_resolve_relay.ts

```diff
@@ -10,7 +10,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   if (Relay && helpers.address.isValid(Relay.address)) {
     log(`using external Relay at ${Relay.address}`)
-  } else if (hre.network.name !== "hardhat") {
+  } else if (!hre.network.tags.local) {
```

Comment:
I am not quite sold on the "local" tag. CI is not local, system tests are not local. What if we do:

```
hre.network.name !== "hardhat" && hre.network.name !== "development"
``` 

instead? I think it's more explicit.

Reply:
[1927937](https://github.com/keep-network/tbtc-v2/pull/365/commits/19279377b8b8e2ddd1fa76acb131e3e97548bce3) Sure, but at the same time when CI uses hardhat's configuration it already has a tag "local". Maybe it doesn't use it but it's present which is not 100% accurate. We can add another tag "CI" to the `hardhat's` network configuration, however, I think it's a broader discussion not for this PR.

Reply:
I prefer tags. Please see my comment in https://github.com/threshold-network/solidity-contracts/pull/108#discussion_r910950449